db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160323170519) do
+ActiveRecord::Schema.define(version: 20160324124129) do
```

Comment:
Yes, but your migration doesn't actually contain anything so I doubt that will help.